### PR TITLE
[CCXDEV-5895] Deploy config for stage (and prod) env

### PR DESCRIPTION
**Pull request by juandspy**
# Description

Add configuration to deploy to OCP.

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: deploy/clowdapp.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            cpu: ${CPU_REQUEST}
+            memory: ${MEMORY_REQUEST}
+          limits:
+            cpu: ${CPU_LIMIT}
+            memory: ${MEMORY_LIMIT}
+        env:
+        - name: ENV_NAME
+          value: ${ENV_NAME}
+    database:
+      name: ccx-data-pipeline
```
Comment (matysek) [inline deploy/clowdapp.yaml:57]: you probably should use here this and drop name and version:
```
sharedDbAppName: ccx-insights-results
```
Then you need to specify dependencies and it should be:
```
dependencies:
  - ccx-insights-results
```

**Review by matysek — CHANGES_REQUESTED** [review]

File: deploy/clowdapp.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  spec:
+    envName: ${ENV_NAME}
+    testing:
+      iqePlugin: ccx
+    jobs:
+    - name: insights-results-aggregator-exporter-job
+      parallelism: 1    
+      completions: 1    
+      activeDeadlineSeconds: ${ACTIVE_DEADLINE_SECONDS} 
+      backoffLimit: ${BACKOFF_LIMIT}
```
[the remark this answers is not in the document]
Comment (juandspy) [reply deploy/clowdapp.yaml]: - parallelism: The pod replicas a job should run in parallel. In this case just one as the process can't be parallelized.
- completions: Successful pod completions are needed to mark a job completed. We just need to export the database once, not more.
- backofflimit: The number of retries for a job. Probably 3 or 4 is a nice number just to make sure the job succeded although sometimes it failed due to connection errors, timeouts, etc.

File: deploy/clowdapp.yaml (hunk 1 above)
Comment (juandspy) [reply deploy/clowdapp.yaml:57]: what's `ccx-insights-results`?

Comment (matysek) [reply deploy/clowdapp.yaml]: @juandspy that is clowdapp name for insights-results-aggregator.

**Review by tisnik — COMMENTED** [review]
please rebase before merge

**Comment by juandspy** [comment]
> please rebase before merge

@tisnik  of course, I'm just throwing commits to test in ephemeral envs :)

**Comment by tisnik** [comment]
please squash these things

**Pull request merged**